script/src/run-dev/index.ts

```diff
@@ -169,7 +169,9 @@ export async function main(
             npmBinCommand({
               name: `${name}:run`,
               command:
-                `while [ ! -f build/index.js ]; do echo "Waiting for build…"; sleep 1; done; nodemon --watch build --delay 1 --exitcrash --exec NODE_ENV=development VX_MACHINE_TYPE=${frontend} node build/index.js`,
+                'while [ ! -f build/index.js ]; do echo "Waiting for build…"; sleep 1; done; ' +
+                'nodemon --watch build --delay 1 --exitcrash --exec ' +
+                'NODE_ENV=development pnpm start',
```

Comment:
I dropped the `VX_MACHINE_TYPE=${frontend}`  here in favor of the provided abstraction for env vars:

https://github.com/votingworks/vxsuite/blob/44baf7fcb5f97ffab5e3584eb977e57b73d369ac/script/src/run-dev/index.ts#L152

-----

I also switched from `node build/index.js` as the command for frontends' dependent services to `pnpm start` so that we have some more control. Every backend's `pnpm start` command is `node ./build/index.js`, with the exception of VxDesign, which uses `node ./build/index.js & node ./build/worker/index.js`

Zooming out, this just makes sure that the background worker is properly started when we run `pnpm start` inside `apps/design/frontend`